### PR TITLE
Feature/Download block

### PR DESCRIPTION
**Pull request by palmenhq**
⬇️

### CODE REVIEW

File: src/blocks/DownloadBlock/AppLinkForm.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      utmParams,
+      defaultBranchLinkOptions,
+    )
+    ;(window as any).branch.sendSMS(
+      phoneNumber,
+      {
+        ...linkOptions,
+        data: {
+          ...(this.props.linkOptions || {}),
+          $custom_sms_text: 'Ladda ner Hedvig-appen: {{ link }}',
```
[the remark this answers is not in the document]
Comment (palmfjord) [reply src/blocks/DownloadBlock/AppLinkForm.tsx]: Fair enough!

[the remark this answers is not in the document]
Comment (palmfjord) [reply src/blocks/DownloadBlock/AppLinkForm.tsx]: actually (after irl discussion) lets push it for later. It'll be hard to do nicely with the "templating" and its better to leave it than to do a semi-good solution that we "should" revisit sometime in the future

File: src/blocks/DownloadBlock/index.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import * as React from 'react'
+import styled from 'react-emotion'
+
+import {
+  ContentWrapper,
+  MOBILE_BP_DOWN,
+  SectionWrapper,
+} from '../../components/blockHelpers'
+import { BaseBlockProps } from '../BaseBlockProps'
+import { TABLET_BP_UP } from '../HeaderBlock/mobile'
```
Comment (isaacSennerholt) [inline src/blocks/DownloadBlock/index.tsx]: Should we keep these in one place? Most of them currently exist in blockHelpers. Do we want to move them into separate files that are more descriptive?

Comment (palmfjord) [reply src/blocks/DownloadBlock/index.tsx]: woops nice catch! Haha i think i just imported it from the wrong place

Comment (palmfjord) [reply src/blocks/DownloadBlock/index.tsx]: (ide magic is dangerous sometimes)

**Pull request merged**